change/@fluentui-react-timepicker-compat-00fd78ea-085d-471c-8c90-5b7cc813e4fb.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31324&commitId=a6397a122c868e2ffc162429ce78e53bf328bbb9&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask.normal.chromium.png|2| Changed|

</details>


  <details><summary><b>Card Converged - Selectable</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card Converged - Selectable.appearance focusable + selectable.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance focusable + selectable.selected.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.click.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.hover.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.normal.chromium.png|0| Added|
|Card Converged - Selectable.appearance selectable - Filled.selected.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>